tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/FeedHenryBotTest.java

```diff
@@ -44,24 +46,24 @@ public static void setup(){
 		fhp.setApiKey(API_KEY);
 		preferenceDialog.ok();
 	}
-	
-	public void importApp(String project, String appName){
+
+	public static void importApp(String project, String appName) {
 		ImportCordovaApplicationWizard w = new ImportCordovaApplicationWizard();
 		w.open();
-		new WaitUntil(new TreeHasChildren(new DefaultTree()), TimePeriod.VERY_LONG);	
+		new WaitUntil(new TreeHasChildren(new DefaultTree()), TimePeriod.VERY_LONG);
 		new DefaultTreeItem(project, appName).setChecked(true);
 		new LabeledCombo("Directory:").setText(WS_PATH);
 		assertTrue(w.isFinishEnabled());
 		w.finish();
-		//wait for shell and enter ssh secret
-		try{
+		// wait for shell and enter ssh secret
+		try {
 			new WaitUntil(new ShellWithTextIsActive("Information"), TimePeriod.LONG);
 			new DefaultText().setText(SSH_SECRET);
 			new PushButton("OK").click();
-		}catch(Exception e){
-			//do nothing
+		} catch (Exception e) {
```

Comment:
You are commenting on an already pushed code, but I agree - it would be better to catch only exception coming from RedDeer. I can fix that.